README.md

```diff
@@ -1,3 +1,3 @@
-# ESP32 API library for [OXRS](https://oxrs.io)
+# ESP32 and PicoW API library for [OXRS](https://oxrs.io)
```

Comment:
Maybe change this title to `API library for...` and then underneath list the devices it supports?

Then it is easy to add to this list in the README.

Ideally we would rename the repo as well, but I think that can wait.